components/newHome/Members/Members.module.scss

```
@@ -0,0 +1,89 @@
+@use "./styles/font";
+@use "../../../styles/position";
+@import "../../../styles/lib";
+
+.container {
+  width: 100vw;
+  height: 100vh;
+  opacity: 1;
+  transition: 0.5s ease;
+
```

```suggestion
position:relative;
z-index: 100 // 100이 정확한 수치는 아니고, 한 번 통일시킬 필요는 있을 듯
```

이렇게 하면 컴포넌트가 off가 아닐 때 맨 위로 올라가게 되어서 다른 컴포넌트가 보이는 예상치 못한 에러를 방지할 수 있지 않을까 합니다. 컴포넌트마다 공통으로 이 설정을 적용해도 좋겠네요